packages/web/components/control/menu-dropdown.tsx

```diff
@@ -37,7 +38,7 @@ export const MenuDropdown: FunctionComponent<Props> = ({
             "rounded-b-lginset": index === options.length - 1,
           }
         )}
-        key={index}
+        key={nanoid()}
```

Comment:
`nanoid` for React component's key is not recommended. Check out the [docs](https://github.com/ai/nanoid#react)